package.json

```diff
@@ -25,6 +25,7 @@
     "datepicker": "git+https://github.com/liqd/datePicker.git",
```

Comment:
shouldn't we remove this? @goapunk I dunno why in mB is still there.

Reply:
yes, we don't need this anymore. I probably forgot to remove it